src/app/(home)/_components/home-feed.tsx

```diff
@@ -26,6 +26,15 @@ function withPagination<T extends MySqlSelect>(qb: T, pageIndex: number) {
   return qb.limit(PAGE_SIZE).offset(pageIndex * PAGE_SIZE);
 }
 
+const getTotalPages = unstable_cache(
+  async () => {
+    const totalRecords = await db.select().from(wotds).count();
```

Comment:
count() is not a valid function, please see getCachedDefinitionCount() in the Statistics component to see how this can be done correctly